test/Dapr.Actors.Test/Runtime/ActorManagerTests.cs

```diff
@@ -17,7 +17,8 @@ public sealed class ActorManagerTests
         private ActorManager CreateActorManager(Type type, ActorActivator activator = null)
         {
             var registration = new ActorRegistration(ActorTypeInformation.Get(type));
-            return new ActorManager(registration, activator ?? new DefaultActorActivator(), JsonSerializerDefaults.Web, NullLoggerFactory.Instance, ActorProxy.DefaultProxyFactory, new DaprHttpInteractor());
+            var interactor = new DaprHttpInteractor(clientHandler: null, "http://localhost:3500", apiToken: null);
```

Comment:
Eventually we'll get the client APIs out of all of these tests. We need to create a similar abstraction for the state manager.